tests/test_distributed/test_gpu_with_dask/test_gpu_with_dask.py

```diff
@@ -572,6 +573,25 @@ def test_with_asyncio(local_cuda_client: Client) -> None:
     assert isinstance(output["history"], dict)
 
 
+def test_invalid_nccl(local_cuda_client: Client) -> None:
```

Comment:
Add a test to make sure NCCL is not loaded when distributed training is not used.